### ISSUE TITLE
Specify different resources per worker in the hostfile

### ISSUE THREAD

**Issue by jmontoyam** [dask/distributed#870]
Hi,

I'm a new dask-distributed user. I would like to setup the network using the dask-ssh command and the corresponding hostfile:

dask-ssh --hostfile hostfile.txt

Let's say the workers have different resources constraints, therefore, I would like
to specify different resources per worker in the hostfile, for instance:

10.121.0.1 --nprocs 1 --nthreads 4
10.121.0.2 --nprocs 1 --nthreads 2
10.121.0.3 --nprocs 1 --nthreads 1
10.121.0.4 --nprocs 1 --nthreads 8
10.121.0.5 --nprocs 1 --nthreads 2
10.121.0.6 --nprocs 1 --nthreads 1
10.121.0.7 --nprocs 1 --nthreads 1
10.121.0.8 --nprocs 1 --nthreads 16
10.121.0.9 --nprocs 1 --nthreads 4
10.121.0.10 --nprocs 1 --nthreads 1

Can I specify different resources per worker in the hostfile and then
call dask-ssh --hostfile hostfile.txt?

Thank you very much for your help.

**Comment by mrocklin** [comment]
Seems like a reasonable thing to add.  You would need to parse keywords a
bit and then change around some code in distributed/deploy/ssh.py.  Is this
something you're interested in contributing?

**Comment by jmontoyam** [comment]
Yes!, sure!, I would like to make my first contribution to this amazing project!, but please be patient with me if  I ask you too many questions, I'm a beginner dask-distributed user :-).
